icons/eraser.svg

```diff
@@ -9,7 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m7 21-4.3-4.3c-1-1-1-2.5 0-3.4l9.6-9.6c1-1 2.5-1 3.4 0l5.6 5.6c1 1 1 2.5 0 3.4L13 21" />
-  <path d="M22 21H7" />
-  <path d="m5 11 9 9" />
+  <path d="M21 21H8a2 2 0 0 1-1.42-.587l-3.994-3.999a2 2 0 0 1 0-2.828l10-10a2 2 0 0 1 2.829 0l5.999 6a2 2 0 0 1 0 2.828L12.834 21" />
```

Comment:
See a lot of extra decimals. Can't we optimize it even more?

Reply:
`1.414` is the correct number as we rotate things here by 45°, the actual number is `2^2`/`1.4142135623730951`.

<img width="511" alt="billede" src="https://github.com/user-attachments/assets/94d1d3e9-c256-4d5c-bad1-fdf20edb1d5a" />